### PR TITLE
Adds screwdriver messages for sleepers

### PR DESCRIPTION
**Pull request by 81Denton**
:cl: Denton
spellcheck: Sleepers now show a message if players try to unscrew the maintenance hatch while they're occupied or open. Fixed typos in sleeper/organ harvester messages.
tweak: Sleepers can now be pried open with crowbars.
tweak: You can open and close sleepers by alt-clicking them.
/:cl:

### CODE REVIEW

**Comment by PKPenguin321** [comment]
User feedback for stuff like this is so good. Thanks

File: code/game/machinery/Sleeper.dm [hunk 1]
```diff
@@ -93,6 +93,12 @@
 	if(!state_open && !occupant)
```
Comment (ninjanomnom) [inline code/game/machinery/Sleeper.dm]: This block can just be moved to below the checks you added now

File: config/admins.txt [hunk 2]
```diff
@@ -135,7 +135,7 @@ octareenroon91 = Game Master
 Serpentarium = Game Master
 Averagejoe82 = Game Master
 The Dreamweaver = Game Master
-88Naoki = Game Master
```
Comment (ninjanomnom) [inline config/admins.txt]: No removing other people

**Review by ninjanomnom — CHANGES_REQUESTED** [review]

File: config/admins.txt (hunk 2 above)
Comment (81Denton) [reply config/admins.txt]: I recently changed my ckey from 88Naoki to Denton-30. @imsxz can confirm it

File: code/game/machinery/Sleeper.dm [hunk 3]
```diff
@@ -90,6 +90,12 @@
 	close_machine(target)
 
 /obj/machinery/sleeper/attackby(obj/item/I, mob/user, params)
+	if(state_open)
+		to_chat(user, "<span class='warning'>[src] must be closed to [panel_open ? "close" : "open"] its maintenance hatch!</span>")
+		return
+	if(occupant)
+		to_chat(user, "<span class='warning'>[src] is currently occupied!</span>")
+		return
 	if(!state_open && !occupant)
```
Comment (ninjanomnom) [inline code/game/machinery/Sleeper.dm]: There is no point where this if will ever fail

**Review by ninjanomnom — DISMISSED** [review]

File: code/game/machinery/Sleeper.dm (hunk 3 above)
Comment (81Denton) [reply code/game/machinery/Sleeper.dm]: I'm not sure how to fix it, with `else return..()`?

Comment (ninjanomnom) [reply code/game/machinery/Sleeper.dm]: You already return if either state_open or occupant are truthy. By the time it gets here the code will always run inside this block so you may as well remove the if check.

**Review by ShizCalev — CHANGES_REQUESTED** [review]

File: code/game/machinery/Sleeper.dm (hunk 3 above)
Comment (ShizCalev) [reply code/game/machinery/Sleeper.dm]: see https://github.com/tgstation/tgstation/blob/12ae904bf1006102e8c868c1236e2a7a650b2f42/code/game/machinery/harvester.dm#L133-L145

Comment (81Denton) [reply code/game/machinery/Sleeper.dm]: Alright, that's doable

**Comment by 81Denton** [comment]
Okay, I replaced all attackby() procs with tool acts and tweaked sleepers so you can A) pry them open with crowbars and B) open/close them by alt-clicking.
Travis is acting up, let me restart it.

File: code/game/machinery/Sleeper.dm [hunk 4]
```diff
@@ -93,17 +93,36 @@
...
 
-/obj/machinery/sleeper/attackby(obj/item/I, mob/user, params)
-	if(!state_open && !occupant)
-		if(default_deconstruction_screwdriver(user, "[initial(icon_state)]-o", initial(icon_state), I))
-			return
-	if(default_change_direction_wrench(user, I))
+/obj/machinery/sleeper/screwdriver_act(mob/living/user, obj/item/I)
+	. = TRUE
+	if(..())
 		return
```
Comment (ShizCalev) [inline code/game/machinery/Sleeper.dm]: TRUE

File: code/game/machinery/Sleeper.dm [hunk 5]
```diff
@@ -93,17 +93,36 @@
...
+	if(default_pry_open(I))
+		return TRUE
+	if(default_deconstruction_crowbar(I))
+		return TRUE
+
+/obj/machinery/sleeper/default_pry_open(obj/item/I) //wew
+	. = !(state_open || panel_open || (flags_1 & NODECONSTRUCT_1)) && I.tool_behaviour == TOOL_CROWBAR
+	if(.)
+		I.play_tool_sound(src, 50)
+		visible_message("<span class='notice'>[usr] pries open \the [src].</span>", "<span class='notice'>You pry open [src].</span>")
```
Comment (ShizCalev) [inline code/game/machinery/Sleeper.dm]: remove \the

**Review by ShizCalev — DISMISSED** [review]

**Pull request merged**